### PR TITLE
add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by naolkidanu**
- add a contact form to the portfolio
- styles contact form section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -214,6 +214,26 @@ <h2 class="title-about">About<br>Myself</h2>
...
+            <input type="email" id="email" name="user_email" placeholder="Enter your email" required>
+            <textarea name="message" id="msg" cols="24" rows="6" maxlength = "500" placeholder="Write your message here" required></textarea>
+            <button type="submit" class="btn">Get It Touch</button>
+        <img class="contact-img"  src="img/Ellipse 3.png" alt="cont-img" >
+        <img class="contact-img2"  src="img/Rectangle 4.png" alt="cont-img" >
+        <img class="contact-img3"  src="img/Ellipse 12.png" alt="cont-img" >
+        <img class="contact-img4"  src="img/Rectangle 7.png" alt="cont-img" >
+        <img class="contact-img5"  src="img/Ellipse 5.png" alt="cont-img" >
+        <img class="shape" src="img/Shape.png" alt="shape-img">
+    </form>
```
Comment (Nemwel-Boniface) [inline index.html:233]: - [ ] Everything else looks awesome apart from one small bit.Kindly note that there is a spacing around your entire footer and form section which is present in your design but is not present in the Figma design. Kindly ensure that you remove the margin/padding that is around your design to match the provided Figma design.

**This is how I am seeing your section right now:**

image

**This is the desired Figma design:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @naolkidanu     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your portfolio so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -214,6 +214,24 @@ <h2 class="title-about">About<br>Myself</h2>
...
+  <footer class="contact-form">
+    <div class="first-container" >
+         <h2 class="contact-head">Contact me </h2>
+        <p class="para-contact">If you have an application you <br> are interested in developing, a <br>  feature that you need built or a <br> project that needs coding. <br> I’d love to help with it </p>
+    </div>
+    <form class="contact-list" action="https://formspree.io/f/xdobrbok" method="post">
+             <input type = "text" id="name"  name="user_name" maxlength = "30" placeholder="Enter Your name"  required>
+            <input type="email" id="email" name="user_email" placeholder="Enter your email" required>
+            <textarea name="message" id="msg" cols="24" rows="6" maxlength = "500" placeholder="Write your message here" required></textarea>
+            <button type="submit" class="btn">Get It Touch</button>
```
Comment (Nyame-Wolf) [inline index.html:226]: [OPTIONAL] Kindly note that your button's color is different from the figma design. Its good practice to have the design match figma desin
This is your buttons color
Screenshot 2022-09-28 at 12 39 05

This is the color in the figma design
Screenshot 2022-09-28 at 12 43 00

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @naolkidanu  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by naolkidanu** [comment]
Thank you for supportive comment and review.

**Pull request merged**